vg/vgsvg/vgsvg.go

```diff
@@ -9,12 +9,13 @@ package vgsvg
 import (
 	"bufio"
 	"bytes"
-	"code.google.com/p/plotinum/vg"
 	"fmt"
-	svgo "github.com/ajstarks/svgo"
 	"image/color"
 	"io"
 	"math"
+
+	svgo "github.com/ajstarks/svgo"
```

Comment:
Maybe in a follow-up drop the svgo import rename and use svg? There is no collision.